systemtest/src/test/java/io/strimzi/systemtest/StrimziUpgradeST.java

```diff
@@ -268,6 +269,10 @@ void tearDownEnvironmentAfterEach() {
         deleteNamespaces();
     }
 
+    @Override
+    void tearDownEnvironmentAfterAll() {
+    }
```

Comment:
You might want to consider a comment about why this is overridden to do nothing.

Reply:
Thanks, added comment